.ci.yaml

```diff
@@ -93,48 +93,32 @@ targets:
       version_file: flutter_master.version
 
   - name: Linux dart_unit_test_shard_1 master
```

Comment:
Do you want to change the name to reflect `web`: `Linux web_dart_unit_test_shard_1 master`

Reply:
This was me hacking the web version into the existing config so I could run in CI without mucking around with `led`. The non-draft version once I have things passing will be new, bringup connfigs.